pallets/phala/src/mining.rs

```diff
@@ -149,7 +345,16 @@ pub mod pallet {
 					challenge_time,
 					iterations,
 				} => {
-					todo!("TODO(wenfeng):");
+					if let Some(binding_miner) = WorkerBinding::<T>::get(&worker_pubkey) {
```

Comment:
Move to GK to handle Heartbeat for unresponsive worker

Reply:
Should we wait the GK side stuff done?